### ISSUE TITLE
archive `kubeflow/fate-operator` repository

### ISSUE THREAD

**Issue by thesuperzapper** [kubeflow/fate-operator#37]
This repository has not had a code commit since [`Jun 2, 2021`](https://github.com/kubeflow/fate-operator/commits/master), therefore I propose we archive it to reflect that it is no longer in development.

**Comment by andreyvelich** [comment]
Today during community call we discussed if Fate Operator should be archived.
@kubeflow/wg-training-leads @tenzen-y @kuizhiqing @zw0610 @owlet42 @LaynePeng @jiahaoc1993 Do we want to adopt Fate Operator to Kubeflow Training Operator ?
Currently Kubeflow doesn't have any story around Federated Learning and it might be nice to see features around this area.

I can see that FATE project has KubeFate component: https://github.com/FederatedAI/KubeFATE.
@owlet42 @jiahaoc1993 @LaynePeng @hainingzhang Should we have a discussion around collaboration between FATE and Kubeflow communities to see if there are any benefits that Kubeflow can provide to adopt running FATE on Kubernetes ?

I am looking forward to your suggestions!

cc @jbottum @zijianjoy @james-jwu @akgraner

**Comment by andreyvelich** [comment]
It would be nice if someone from FATE community could join our next [Kubeflow Training and AutoML WG Community Call](https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64/edit#heading=h.1yazr7lkeexq) on this Wednesday at September 6th 5pm UTC (10am PST), so we can discuss topics around Fate Operator in Kubeflow.

What do you think @LaynePeng @nemirorox @talkingwallace @owlet42 @hainingzhang ?

**Comment by tenzen-y** [comment]
> Do we want to adopt Fate Operator to Kubeflow Training Operator ?

I don't have strong motivations to consolidate fate-controller with training-operator.

**Comment by kuizhiqing** [comment]
> I don't have strong motivations to consolidate fate-controller with training-operator.

+1  archive it would be OK.

**Comment by LaynePeng** [comment]
I need to confirm if a project has been achieved, it still can be accessed, downloaded, and cloned, right? If yes, we are okay to archive the fate-operator project.

**Comment by andreyvelich** [comment]
>  it still can be accessed, downloaded, and cloned, right? 

That's correct, similar to kfctl: https://github.com/kubeflow/kfctl, users can access and fork it.

@LaynePeng Would you like to attend any of our Kubeflow community meetings to discuss potential collaboration between KubeFate and Kubeflow projects ?

**Comment by LaynePeng** [comment]
So it looks good because we still have some users of the fate-operator. 

and for attending the community meeting, absolutely good. I will check the community meeting calendar and get back.

**Comment by andreyvelich** [comment]
Thank you @LaynePeng!
@jbottum Please can you allocate some time during one of our Kubeflow community meetings to discuss FATE & Kubeflow potential integrations.

**Comment by jbottum** [comment]
I will put it on the agenda

**Comment by andreyvelich** [comment]
Thank you, Josh!
@LaynePeng Kubeflow Community call is every Tuesday at 8:00am PST: https://docs.google.com/document/d/1Wdxt1xedAj7qF_Rjmxy1R0NRdfv7UWs-r2PItewxHpE/edit#heading=h.7ev8bj8hjrb9

**Comment by andreyvelich** [comment]
Hi @LaynePeng @nemirorox @talkingwallace @owlet42 @hainingzhang, please can you attend the next Kubeflow community call on Jan 9th: https://docs.google.com/document/d/1Wdxt1xedAj7qF_Rjmxy1R0NRdfv7UWs-r2PItewxHpE/edit ?
We need to make final discussion whether we want to archive Fate Operator repo.

**Comment by andreyvelich** [comment]
Hi Folks, do we have any updates here @LaynePeng @nemirorox @talkingwallace @owlet42 @hainingzhang ?
We are planning to archive this repo in 2 weeks if we don't hear back from you.
cc @kubeflow/kubeflow-steering-committee
